src/NuGet.Clients/NuGet.SolutionRestoreManager/xlf/Resources.fr.xlf

```diff
@@ -22,6 +22,11 @@
         <target state="translated">Le paquet '{0}' {1} n’a pas de version exacte comme '[1.0.0]'. Seules les versions exactes sont autorisées avec PackageDownload.</target>
         <note>0 - The package that does not have an exact version. Do not translate 'PackageDownload'. 1 - the version string that's not exact.</note>
       </trans-unit>
+      <trans-unit id="InfoBar_HyperlinkGHCopilotDocs">
+        <source>How to fix with GitHub Copilot</source>
+        <target state="new">How to fix with GitHub Copilot</target>
```

Comment:
Do we want to wait for this to be translated in dev, and then include the translations in the cherry-pick?

Otherwise we'll either have english text for all customers, or we'll need to do another QB insertion once the translations come in.

Reply:
@martinrrm may know more. It's been about 2 weeks so I'm not sure how long this process takes or how branches work.

Reply:
Loc CI was stuck, I'll cherry-pick the relevant strings into this PR once it's merged...